source/aggregation-pipeline-builder.txt

```diff
@@ -370,6 +374,40 @@ To create a view from your pipeline results:
 
 .. include:: /includes/fact-create-view-from-pipeline.rst
 
+.. _refresh-document-counts-after-deletes:
+
+Refreshing Document Counts after Delete Operations
+--------------------------------------------------
+
+When running an aggregation pipeline, the :guilabel:`count results` 
```

Comment:
Consider getting rid of the participle. Also, we don't want users to think that results are inaccurate when the number displayed is what's actually inaccurate. Maybe something like this:

After a delete operation in an aggregation pipeline, the :guilabel:`count results` value needs to be refreshed. To update the document count after a delete: